server/presenters/webhook.ts

```diff
@@ -0,0 +1,34 @@
+import { WebhookDelivery, WebhookSubscription } from "@server/models";
+import { Event } from "@server/types";
+
+interface WebhookProps {
+  event: Event;
+  delivery: WebhookDelivery;
+  subscription: WebhookSubscription;
+  payload: { model: unknown; id?: string };
+}
+
+export interface WebhookPresentation {
+  id: string;
+  webhookSubscriptionId: string;
+  teamId: string;
+  event: string;
+  payload: { model: unknown; id?: string };
+  createdAt: Date;
+}
```

Comment:
This is a type that represents the final shape that gets sent to each webhook url